go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -25,3 +31,19 @@ func IsHostEndpoint(endpointIP string) bool {
 	}
 	return true
 }
+
+func getExternalIDsForLoadBalancer(service *v1.Service, netInfo util.NetInfo) map[string]string {
+	nsn := ktypes.NamespacedName{Namespace: service.Namespace, Name: service.Name}
+
+	role := types.NetworkRoleDefault
+	if !netInfo.IsDefault() {
+		role = types.NetworkRolePrimary
```

Comment:
Actually yes, I can distinguish between default, primary and secondary, so that when we want to run the services controller from a secondary network, the code is ready.